lib/main_common.pm

```diff
@@ -373,21 +373,20 @@ sub load_yast2_ui_tests {
     loadtest "console/yast2_ntpclient";
     loadtest "console/yast2_tftp";
     loadtest "console/yast2_vnc";
-    # TODO https://progress.opensuse.org/issues/20200
+    # product issue #bcs1049433
```

Comment:
s/#bcs/bsc#/
But I suggest to keep the TODO and the ticket URL as we should not keep test modules disabled in general. Better soft fail within the test module